data/clients.json

```diff
@@ -165,7 +165,8 @@
 		},
 		{
 			"name": "acme-client",
-			"url": "https://kristaps.bsd.lv/acme-client/",
+			"url": "https://cvsweb.openbsd.org/cgi-bin/cvsweb/src/usr.sbin/acme-client/",
+			"acme_v2": "OpenBSD >= 6.6",
 			"category": "C"
 		},
```

Comment:
Maybe the description could specify it's for OpenBSD  ?

Reply:
(or change the name to "OpenBSD acme-client"?)

Reply:
@tdelmas 
I think both changes make sense. This program is no longer maintained as a standalone package, so it probably won't be available anywhere else.

Would you like to squash the commit with fixup? What is your merging strategy?

Reply:
No need to squash, (it will be done during the merge, so history of master stays clear, and history of the PR is not rewritten 😄 ) , just add commits with the changes, thanks!